src/opnsense/mvc/app/models/OPNsense/Core/ACL/ACL.xml

```diff
@@ -365,7 +365,8 @@
     <page-wizard-system>
         <name>System Setup Wizard</name>
```

Comment:
Menu says System: Wizard, I wonder if can bury it in System: Settings

Reply:
I don't have a strong preference, can change the acl description or the menu item.

Reply:
system: settings or system: configuration, decide after merge